### DISCUSSION TITLE
建议在小工具中增加同步至知乎、csdn，以及从知乎、csdn拉取功能

### DISCUSSION THREAD

**Discussion by NbTOne** [halo-dev/halo#6644] [Ideas]
### 你当前使用的版本

1.5.3

### 描述一下此特性

![图片](https://user-images.githubusercontent.com/39721050/174012291-dd9d0e7b-d812-4107-aba4-9223e97fe278.png)

### 附加信息

/assign

**Comment by ruibaby** [comment]
感谢你的建议，但目前不打算在 1.x 添加这样的功能。建议等到 2.0 发布之后，通过插件来实现这样的功能。如果感兴趣，可以关注我们的开发动态。

https://github.com/halo-dev/halo/pulls?q=is%3Apr+milestone%3A2.0+is%3Aclosed

https://github.com/halo-dev/rfcs/blob/main/plugin/pluggable-design.md
